docs/semgrep-secrets/getting-started.md

```diff
@@ -48,6 +45,15 @@ After you've enabled Semgrep Secrets, you can:
 * Wait for your scheduled Semgrep full scan
 * Open a pull request or merge request and wait for Semgrep to scan the branch automatically
 
+## Configure files to ignore
+
+Semgrep Secrets will scan files even if they are specified in a local `.semgrepignore`, since secrets can often be found in files that are not relevant for code scanning. To specify files to ignore in Semgrep Secrets as well:
```

Comment:
```suggestion
Semgrep Secrets scans all files, even those specified in a local `.semgrepignore` file, since secrets can often be found in files that are not relevant for code scanning. To specify files that Semgrep Secret should ignore:
```

Reply:
Semgrep Secrets :)

I'll commit something like both of these changes but I want to make the additional fixes as well